OKTA-750492 SCIM deprecation (#5010)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/concepts/iam-overview-architectural-factors/index.md

Identity or IAM is a collection of capabilities/building blocks essential for enabling your overall architecture and user experience. Your IAM solution's success depends on meeting some strict architectural requirements. 

This guide details specific architectural capabilities for making your solution as robust as possible for your org and as seamless as possible for your users. These capabilities are highly complex to build and integrate on your own if they aren't a core competency. It's much more efficient and effective to consider them in your high-level design as early as possible in your solution's life cycle. 

Failure to meet any of the capabilities can lead to the following:

- Reduced availability
- Higher support costs
- Business disruptions
- Erosion of trust
- Customer and user dissatisfaction
- Damage to your reputation
- Expensive breaches and lost business, in the worst case scenario

> > **Note:** This list isn't exhaustive, and strategies for any of them are too numerous and complex to go into here. Okta intends to identify some of the critical Non-Functional Requirements (NFRs) for your IAM solution and to illustrate a few possible strategies to address them.

Reviewer: ```suggestion
> **Note:** This list isn't comprehensive. Strategies for breaches are too numerous and complex. Okta intends to identify some of the critical Non-Functional Requirements (NFRs) for your IAM solution and to illustrate a few possible strategies to address them.
```
